desktop/apps/auction/components/Layout.js

```diff
@@ -76,9 +119,12 @@ const mapStateToProps = (state) => {
   const showFollowedArtistsRail = Boolean(!isMobile && state.artworkBrowser.showFollowedArtistsRail)
   const showMyActiveBids = Boolean(me && me.bidders.length && is_open && !is_live_open)
   const showFooter = Boolean(!isMobile && articles.length || !showFilter)
+  const buyNowSaleArtworks = get(state.app.auction.toJSON(), 'promoted_sale.sale_artworks', [])
 
   return {
     associatedSale: associated_sale,
+    buyNowSaleArtworks: buyNowSaleArtworks.concat(buyNowSaleArtworks),
```

Comment:
Is this to support additional fetching when other pages are requested?

Reply:
Oh I need to add a FIXME here -- this was just so i could work with paging without having to actually create a bunch of sale artworks.